cmd/agent/subcommands/diagnose/command.go

```diff
@@ -14,6 +14,7 @@ import (
 	"sort"
 	"strings"
 
+	workloadfilter "github.com/DataDog/datadog-agent/comp/core/workloadfilter/def"
```

Comment:
Nit: Please move this import lower so that it's with the other "github.com/DataDog/datadog-agent" imports.